src/keycloak/keycloak-js/keycloak-authz.ts

```
@@ -0,0 +1,432 @@
+}
+
+export interface Uma2Configuration {
+    token_endpoint: string;
+    rpt_endpoint?: string;
+    [key: string]: unknown;
+}
+
+export class KeycloakAuthorization {
+    rpt: string | null = null;
+    config: Uma2Configuration | undefined;
+    then: KeycloakAuthorizationPromise["then"] | undefined;
```

Biome (2.1.2)

[error] Do not add then to a class.



(lint/suspicious/noThenProperty)